### ISSUE TITLE
Helpers in header / implementation

### ISSUE THREAD

**Issue by eric-lindau** [alecjacobson/computer-graphics-raster-images#74]
Are we allowed to create e.g. a "common.h/common.cpp" with common utilities, as long as it compiles/works correctly?

**Comment by Jack-XHP** [comment]
no, you should put all your code in cpp files in the handout's src folder.
